python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -184,7 +184,7 @@ class Job(GangaObject):
 
     _schema = Schema(Version(1, 6), {'inputsandbox': FileItem(defvalue=[], typelist=['str', 'Ganga.GPIDev.Lib.File.File.File'], sequence=1, doc="list of File objects shipped to the worker node "),
                                      'outputsandbox': SimpleItem(defvalue=[], typelist=['str'], sequence=1, copyable=_outputfieldCopyable(), doc="list of filenames or patterns shipped from the worker node"),
-                                     'info': ComponentItem('jobinfos', defvalue=None, doc='JobInfo '),
+                                     'info': ComponentItem('jobinfos', defvalue=JobInfo(), doc='JobInfo '),
```

Comment:
I'm not sure that this should be needed. The default value of `defvalue` is `None` and that should trigger the logic in `Schema._getDefaultValueInternal` to load a default-constructed object of the default plugin for the category. If this isn't working then it suggests a deeper problem.

Alternatively setting it to `defvalue='JobInfo'` should work but it *shouldn't* be needed.